src/cmd/create.go

```diff
@@ -265,28 +265,6 @@ func createContainer(container, image, release string, showCommandToEnter bool)
 	logrus.Debugf("%s canonicalized to %s", currentUser.HomeDir, homeDirEvaled)
 	homeDirMountArg := homeDirEvaled + ":" + homeDirEvaled + ":rslave"
 
-	bootMountFlags := "ro"
-	isBootReadWrite, err := isPathReadWrite("/boot")
```

Comment:
I wonder if there are host operating systems out there where we will have to check the read/write status of `/`?

I guess it's good to keep it simple unless we know for sure. :)